packages/state-transition/src/block/processAttestationPhase0.ts

```diff
@@ -93,7 +93,7 @@ export function validateAttestation(
       throw new Error(`AttestationData.index must be zero: index=${data.index}`);
     }
     const attestationElectra = attestation as electra.Attestation;
-    const committeeBitsLength = attestationElectra.committeeBits.bitLen;
+    const committeeBitsLength = attestationElectra.committeeBits.getTrueBitIndexes().length;
```

Comment:
@g11tech is this correct?

Reply:
The `committeeBits` is a bit vector so the `bitLen` will always remain fixed which is `MAX_COMMITTEES_PER_SLOT`. Hence if a chain is starting with initial validators `less than MAX_COMMITTEES_PER_SLOT` the attestation check will never pass. 

So we actually need to get how many comittees actually participatd using `getTrueBitIndexes`.

Reply:
So something like this:
```
    const committeeIndices = attestationElectra.committeeBits.getTrueBitIndexes();
    if (committeeIndices.length === 0) {
        throw Error("Should have at least one committee bit set");
    } else {
        const lastCommitteeIndex = committeeIndices[committeeIndices.length - 1];
        if (lastCommitteeIndex >= committeeCount) {
            throw new Error("Committee index exceeds committee count");
         }
    }
```

This is beyond the scope of this PR, happy to fix this in another PR. Maybe add a todo here for now